src/main/java/com/deque/axe/android/rules/hierarchy/base/TouchSize.java

```diff
@@ -43,35 +51,22 @@ public String runRule(AxeProps axeProps) {
     final long height = frame.height();
     final long width = frame.width();
 
-    final int screenHeight = axeProps.get(AxeProps.Name.SCREEN_HEIGHT, Integer.class);
```

Comment:
Don't remove these props from AxeDevice. Even though we don't use them, we can't guarantee others are not.

Reply:
This does not remove `SCREEN_HEIGHT` prop, I am still saving the screen height in it inside the `setup` method. So as a user who is getting `SCREEN_HEIGHT` value can still access it because it will be stored in setup method.

Reply:
That's entirely my bad, I misread this class as AxeDevice.